salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -9,9 +9,14 @@ edx:
           SubnetId: {{ subnetids[id_num % num_subnets] }}
           SecurityGroupId: {{ securitygroupid }}
       tag:
+        business_unit: {{ business_unit }}
         environment: {{ environment_name }}
+        purpose_prefix: {{ purpose_prefix }}
       grains:
+        business_unit: {{ business_unit }}
         environment: {{ environment_name }}
+        {% set modified_purpose = purpose + '-' + app_type %}
```

Comment:
This needs to reference `purpose_prefix`. As it is right now it will throw an undefined variable exception because `purpose_prefix` is the variable getting set via the `context` in the orchestrate state.